be/src/http/action/config_action.cpp

```diff
@@ -118,7 +118,6 @@ void ConfigAction::handle_update_config(HttpRequest* req) {
             std::string status(s.ok() ? "OK" : "BAD");
             rapidjson::Value result;
             result.SetObject();
-            rapidjson::Value(key.c_str(), key.size(), results.GetAllocator());
```

Comment:
is here no problems before ?

Reply:
No, IIUC it just creates an useless JSON value and drops it immediately.

Reply:
No, IIUC it just creates an useless JSON value and drops it immediately.